ui/component/fileDownloadLink/view.jsx

```diff
@@ -41,19 +44,20 @@ function FileDownloadLink(props: Props) {
     hideOpenButton = false,
     hideDownloadStatus = false,
     streamingUrl,
+    contentRestrictedFromUser,
+    isProtectedContent,
   } = props;
 
   const [didClickDownloadButton, setDidClickDownloadButton] = useState(false);
   const fileName = claim && claim.value && claim.value.source && claim.value.source.name;
 
-  // @if TARGET='web'
+  // initiate download when streamingUrl is available
   React.useEffect(() => {
     if (didClickDownloadButton && streamingUrl) {
-      webDownloadClaim(streamingUrl, fileName);
+      webDownloadClaim(streamingUrl, fileName, isProtectedContent);
       setDidClickDownloadButton(false);
     }
   }, [streamingUrl, didClickDownloadButton, fileName]);
-  // @endif
 
   function handleDownload(e) {
     setDidClickDownloadButton(true);
```

Comment:
I'm in `localhost | pk_test` and I just subscribed.  I can't seem to download the binary file.
Is it because of localhost, caching, or something else?

<img width="1261" alt="image" src="https://user-images.githubusercontent.com/64950861/196741459-854c7bf8-5320-45e2-95c4-cab930d240f3.png">